src/parser/utils.rs

```diff
@@ -95,6 +95,36 @@ impl InfixOp {
     }
 }
 
+/// Turn record destructors using literal chunks only into static accesses
```

Comment:
```suggestion
/// Turn dynamic accesses using literal chunks only into static accesses
```
While destructor is technically correct, I don't think we use it elsewhere in the codebase.